enterprise/docker-compose.yml

```diff
@@ -2,14 +2,21 @@
 # Please take a look at the README at the top of this repo or the regular docs for more info.
 
 services:
+  # MongoDB: https://hub.docker.com/_/mongo/
   mongodb:
-    image: "mongo:5.0"
+    image: "mongo:6.0.18"
+    ports:
+      - "27017:27017"   
```

Comment:
This exposes the MongoDB port on all IP addresses. Worst case, on the internet. So we either remove the `ports` mapping or use `- "127.0.0.1:27017:27017"` to only expose the port on localhost on the host machine.